src/domUpdates.js

```diff
@@ -71,35 +72,40 @@ const domUpdates = {
       </section>`
     })
   },
-  displayTripDetails(trip) {
-    console.log(trip);
+  displayTripDetails(trip, isNew) {
     let tripDetailsPopup = document.querySelector(".trip-details");
     tripDetailsPopup.style.display = "inline";
     tripDetailsPopup.innerHTML = ''
-    this.displayTripDetailsHeader(trip, tripDetailsPopup)
-    this.displayTripDetailsImage(trip)
-    tripDetailsPopup.insertAdjacentHTML("beforebegin", "<section id='overlay'></div>");
+    this.displayTripDetailsHeader(trip, tripDetailsPopup, isNew)
+    tripDetailsPopup.insertAdjacentHTML("beforebegin", "<section id='overlay'></section>");
   },
-  displayTripDetailsHeader(trip, tripDetailsPopup) {
+  displayTripDetailsHeader(trip, tripDetailsPopup, isNew) {
     let range = trip.dateRange()
     let startDate = moment(new Date(range[0])).format('MM/DD/YYYY')
     let endDate = moment(new Date(range[range.length - 1])).format('MM/DD/YYYY')
-    tripDetailsPopup.innerHTML = ``
-    tripDetailsPopup.innerHTML =
-    `<button id="exitTripDetails" class="exitTripDetails">X</button>
-      <h3 id="tripDetailsDest">${trip.destinationData.destination}</h3>
+    tripDetailsPopup.innerHTML = `<img src="${trip.image}" class="detailPhoto" id="trip-${trip.id}" alt="${trip.alt}" title="trip">`
+    tripDetailsPopup.innerHTML +=
+    `<h3 id="tripDetailsDest">${trip.destinationData.destination}</h3>
       <h4>Dates : ${startDate} - ${endDate} (${trip.duration} days)</h4>
       <h4>Total Travelers : ${trip.totalTravelers}</h4>
       <h4>Estimated Cost : $${trip.determineTripCost()}</h4>
       <h4>Status : ${trip.status}</h4>`
-  },
-  displayTripDetailsImage(trip) {
-    document.getElementById("tripDetailsDest").style.backgroundImage = `url(${trip.image})`
+    if (isNew) {
```

Comment:
Are you writing over the innerHTML on line 86? If so I would use an `if else` or `else` to cut down on the number of lines and DOM manipulation Javascript has to execute